### PR TITLE
Fixed some commands for local setup

### PR DESCRIPTION
**Pull request by MohitKumar020291**

### CODE REVIEW

**Review by MohitKumar020291 — COMMENTED** [review]
kindly approve some changes

**Comment by MohitKumar020291** [comment]
I do not know why it is showing that merging is blocked

File: website/events/helpers/models.py [hunk 1]
```diff
@@ -1,9 +1,14 @@
+# for local setup
```
Comment (facundobatista) [inline website/events/helpers/models.py:1]: All dependencies are declared in `config/requirements.txt`, used by the Dockerfile that prepares the environment, they should not be duplicated on each `.py`...

File: website/events/helpers/models.py [hunk 2]
```diff
@@ -1,9 +1,14 @@
...
+# pip install django
+# pip install django-extensions
 from django.contrib.auth import get_user_model
 from django.db import models
 from django_extensions.db.models import TimeStampedModel
 from events.middleware import get_current_user
 
-import reversion
+from django.urls import reverse
+Redirect_Variable = reverse()
```
Comment (facundobatista) [inline website/events/helpers/models.py:11]: What is this `Redirect_Variable` for?

File: website/events/helpers/models.py [hunk 3]
```diff
@@ -40,7 +45,7 @@ class SaveReversionMixin:
     '''Mixin to override save, wrapping with create_revision.
     To better work keep on left on inheritance.'''
     def save(self, *args, **kwargs):
-        with reversion.create_revision():
+        with reverse.create_revision():
```
Comment (facundobatista) [inline website/events/helpers/models.py:48]: Why this change?

**Review by facundobatista — CHANGES_REQUESTED** [review]
Hello! Thanks for the PR! I left a couple of questions... thanks!

File: website/events/helpers/models.py (hunk 3 above)
Comment (MohitKumar020291) [reply website/events/helpers/models.py:48]: again reversion was not working so I changed it.
Hope it explains all questions , thanks for checking out my PR.

Comment (andresramirezmiori) [reply website/events/helpers/models.py:48]: reversion (https://django-reversion.readthedocs.io/en/stable/index.html) is now part of django.urls?.

**Comment by MohitKumar020291** [comment]
Yes , reversion is part of django

File: website/events/helpers/models.py (hunk 3 above)
Comment (facundobatista) [reply website/events/helpers/models.py:48]: But `reverse` does not have a `create_revision` function inside...

Comment (MohitKumar020291) [reply website/events/helpers/models.py:48]: Oh.. , I mistakenly did not checked it . So now , should I make a funtion of  create_revision or keep it the same as previous?

Comment (facundobatista) [reply website/events/helpers/models.py:48]: I still don't know why you want to change this.

**Comment by MohitKumar020291** [comment]
@facundobatista I am closing this pull request. After research I found that it does not make any sense. I deeply apologize for very late reply.
